conf/st2rc.sample.ini

```diff
@@ -22,6 +22,15 @@ password = testpassword
 
 [api]
 url = http://127.0.0.1:9101/v1
+# or for a remote instance
+# url = https://example.com/api/v1
 
 [auth]
 url = http://127.0.0.1:9100/
+# or for a remote instance
```

Comment:
I'll actually confirm it works correctly with the paths I used.

it's a bit annoying those URLs are not consistent, but making them consistent might require code changes :)

Reply:
Confirmed those URLs work correctly with the default nginx config so the same consistent suffix can be used everywhere.